start/src/test/java/org/apache/accumulo/start/classloader/vfs/ContextManagerTest.java

```diff
@@ -61,16 +59,15 @@ static FileSystemManager getVFS() {
     }
   }
 
-  @Before
+  @BeforeEach
   public void setup() throws Exception {
-
     vfs = getVFS();
-    folder1 = tempFolder.newFolder();
-    folder2 = tempFolder.newFolder();
+    folder1 = new File(tempFolder, "folder1");
+    folder2 = new File(tempFolder, "folder2");
```

Comment:
It looks like the tests pass fine like this but it might be a good idea to create a unique directory for each test like you did in AccumuloReloadingVFSClassLoaderTest with `folder1` and `tmpDir`

Reply:
The current code used these folder names. It's probably best to just leave them as-is, since everything still works as expected.

Reply:
The current way that these folders are created is different from before these changes were made. Before, a Junit4 TemporaryFolder was created for each test method and folder1 and folder2 were created inside that per-test-method directory giving them a unique path. Now, with the static TempDir, a single folder is created for all test methods and each test method will create these folders with the same path which may cause problems in certain situations. I guess its fine for now since these tests pass. I was just thinking that it could potentially cause issues in the future if more test methods are added that use these folders.